todo/tests/unit/models/test_task.py

```
@@ -1,3 +1,4 @@
+from typing import List
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)